Car1/src/car1/CarSetup.java

```diff
@@ -0,0 +1,104 @@
+
+package car1;
+
+/**
+ *The CarSetup class contains methods that will be used in the Car1 program.
+ * @author mkavic
+ */
+public class CarSetup {
+   
+   // 
+   private int yearModel; // car's year model
+   private String make; // car's makes
+   private int speed = 0; //car's speed.
+  
+   /**
+    * Constructor
+    * @param yearMod car's year model
+    * @param mak car's make
+    * @param sped car's speed
+    */
+   public CarSetup(int yearMod, String mak, int sped)
+   {
+       yearModel =  yearMod;
+       make = mak;
+       speed = sped;
+   }
+   
+   /**
+    * The setYearModel method stores a value in the yearModel field.
+    * @param yearMod The value to store in yearModel field.
+    */
+  
+   public void setYearModel(int yearMod)
+    {
+        yearModel = yearMod;
+    }
+   
+   /**
+    * The setMake method stores a value in the make field.
+    * @param mak The value to store in make.
+    */
+    
+   public void setMake (String mak)
+   {
+       make = mak;
```

Comment:
generally the pattern using `this` allows these variables to be the same name to help with readability later